ast35/Python/ast.c

```diff
@@ -1447,6 +1469,13 @@ ast_for_arguments(struct compiling *c, const node *n)
                 if (TYPE(ch) == COMMA) {
```

Comment:
This was an if/else that implicitly was "type COMMA or type tfpdef (/ vfpdef?)", but as written I think there's now another case -- the next child could be a type comment. This is basically another case that deserves the exact same error message as the `i+1 >= NCH(n)` condition above leads to, so it should probably just be added to that conditional.

While at it, maybe this `else` should get a comment saying `tfpdef or vfpdef`, or whatever exactly is the right description there.